.github/workflows/check-toc.yml

```diff
@@ -1,8 +1,7 @@
 name: Check
 
 on:
-  workflow_dispatch:
```

Comment:
Why no workflow dispatch?

Reply:
If it is committing to the host repository it should arguably occur on `push`.

Reply:
I'm not getting it yet. You could do a workflow dispatch in your own fork too, right? But I don't care as much so this is ok to remove.

Reply:
Ah you're right. Will re-add.